news/4640-use-grayskull

```diff
@@ -0,0 +1,23 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* `conda-build skeleton` is pending deprecation. `conda-build` still provides the
+  `conda-build skeleton` a.k.a. `conda skeleton` CLI for now, but will remove
+  it eventually.
```

Comment:
```suggestion
* `conda skeleton` is pending deprecation. `conda-build` still provides
  `conda-skeleton` a.k.a. `conda skeleton` CLI for now, but will remove
  it eventually.
```

Reply:
D'oh!

Reply:
At least the CLI plugin system is a single, less confusing way to have more CLI commands.

```
% ls conda-*
conda-build             conda-grayskull         conda-render
conda-convert           conda-greyskull         conda-server
conda-debug             conda-index             conda-skeleton
conda-develop           conda-inspect           conda-verify
conda-env               conda-metapackage
```